### DISCUSSION TITLE
Wazuh 4.6.0 is out?

### DISCUSSION THREAD

**Discussion by T100D** [wazuh/wazuh#20040] [Q&A]
Running 4.5.4 and checking for release notes we saw that 4.6.0 is present in the repositories and so we updated. No problems whatsoever and adjusted the config file running all in a  small setup.

Stil no mention or release notes  that 4.6.0 is out.

Did something went wrong?

Kind greetings,
Arie.

**Comment by lpingree** [comment]
I just upgraded and the gui won't load. So I'd hold off.
